Project.toml

```
@@ -5,7 +5,9 @@ version = "0.1.2-DEV"
 
 [deps]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+TrixiBase = "9a0f1c46-06d5-4909-a5a3-ce25d3fa3284"
 
 [compat]
 Test = "1"
+TrixiBase = "0.1.3"
```

```suggestion
TrixiBase = "0.1.6"
```
TrixiBase.jl v0.1.6 is required with this PR and will be out soon.